src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -23,1615 +23,1657 @@
 package com.intel.genomicsdb;
 
 import com.googlecode.protobuf.format.JsonFormat;
+import com.intel.genomicsdb.model.BaseImportConfig;
+import com.intel.genomicsdb.model.VCFHeaderToFile;
```

Comment:
Difficult to review - what changed in this file?

Reply:
There are changes all over the place... but the most important one is the one related to parallelImport.